### PR TITLE
removed deprecated and nodejs 8 images from e2e tests and README

### PR DESCRIPTION
**Pull request by girishramnani**
**What type of PR is this?**
> Uncomment only one ` /kind` line, and delete the rest.
> For example, `> /kind bug` would simply become: `/kind bug`

 /kind failing-test

**What does does this PR do / why we need it**:
Removes deprecated and nodejs 8 images from e2e tests

**Which issue(s) this PR fixes**:

Fixes https://github.com/openshift/odo/issues/2641

**How to test changes / Special notes to the reviewer**:
CI shouldn't fail

### CODE REVIEW

**Comment by mik-dass** [comment]
```
error: some steps failed:
  * could not run steps: step [release:latest] failed: failed to create or restart release pod: unable to create pod: timed out waiting for the condition
```
/retest

**Comment by girishramnani** [comment]
/hold 
Readme needs an update too @mik-dass - give me a second

File: README.adoc [hunk 1]
```diff
@@ -36,19 +36,6 @@ Existing tools such as `oc` are more operations-focused and require a deep-under
...
-| |https://access.redhat.com/articles/3376841[rhoar-nodejs/nodejs-8]
-|NPM
-
-|
-|https://www.github.com/bucharest-gold/centos7-s2i-nodejs[bucharestgold/centos7-s2i-nodejs]
-|NPM
-
-|
-|https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/nodejs-8-rhel7[rhscl/nodejs-8-rhel7]
-|NPM
```
Comment (mik-dass) [inline README.adoc:51]: Java seems to have moved from the language column to the container image column

Comment (girishramnani) [reply README.adoc:51]: yeah taking a look

Comment (girishramnani) [reply README.adoc:51]: done @mik-dass

Comment (mik-dass) [reply README.adoc:51]: :+1:

**Pull request merged**